### ISSUE TITLE
[gcdt]Managing ENV in gcdt

### ISSUE THREAD

**Issue by Dgadavin** [glomex/gcdt#325]
My purpose is to start managing ENV in gcdt instead of exporting it like environment variables. It will be more explicit way to have setting environment as a docopt command. 
For example:
`$ gcdt setenv <ENV>`

Also, we can add some help info to make this process more clearly for common user.

**Comment by markfink** [comment]
I do not think this will work since `gcdt` does not have state unless it is stored in config files. One thing that would be  possible is to add an `--env=dev` option to the relevant commands (probably all). But to me it is unclear if this solves your usecase.

To my knowledge most users do this instead:
```
ENV=dev kumo deploy
```
so after all not that big an improvement...

**Comment by Dgadavin** [comment]
`$ kumo deploy --env=dev`
I think such way much more clear for users because they set `env` and understand for what they do this.
For situation where `we have an ENV variable and --env option where values differ.` I think if users will not know about ENV variable and use only `--env=dev` we will partially avoid this situation.

**Comment by jbrandstetter** [comment]
We actually had --env some time ago and removed it because it was unclear or at least confusing what happened if those two differ. So for the sake of simplicity and to adhere to 12 factor app rules we decided to stick with env vars. I actually do think that we can assume people understand env variables if we let them deal with highly complex cloud infrastructure ;-) On the other hand I can see that setting the env via a gcdt command (could also be kumo setenv, ramuda setenv etc) adds to the clarity of the current context. So I'd propose to stick to env variables but introduce a new command "setenv" available for each tool that sets the ENV variable to the given value.
